src/Selenium2Driver.php

```diff
@@ -222,25 +222,29 @@ protected function withSyn()
     /**
      * Creates some options for key events
      *
-     * @param string $char     the character or code
-     * @param string $modifier one of 'shift', 'alt', 'ctrl' or 'meta'
+     * @param string|int  $char     the character or code
+     * @param string|null $modifier one of 'shift', 'alt', 'ctrl' or 'meta'
      *
      * @return string a json encoded options array for Syn
      */
-    protected static function charToOptions($char, $modifier = null)
+    protected static function charToOptions($char, $modifier)
```

Comment:
turning a argument into a mandatory one in a public or protected method is a BC break.

Reply:
I belive the default value here is not needed, but I reverted this change, it is not needed